tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -748,7 +748,7 @@ TEST (nnstreamer_nnfw_mlapi, invoke_pipeline_00)
   size_t data_size;
 
   gchar *test_model;
-  guint *sink_called_cnt  = (guint *) g_malloc (sizeof (guint));
+  guint *sink_called_cnt = (guint *) g_malloc0 (sizeof (guint));
```

Comment:
Is there any reason to use a pointer of `guint`? What happens if just using `guint sink_called_cnt = 0;`

Reply:
to avoid reference of local variable in callback.